Sources/Sentry/SentryFramesTracker.m

```diff
@@ -99,10 +88,22 @@ - (void)displayLinkCallback
         return;
     }
 
+    // Calculate the actual frame rate as pointed out by the Apple docs:
+    // https://developer.apple.com/documentation/quartzcore/cadisplaylink?language=objc The actual
+    // frame rate can change at any time by setting preferredFramesPerSecond or due to promotion
```

Comment:
```suggestion
    // frame rate can change at any time by setting preferredFramesPerSecond or due to ProMotion
```

l:To avoid confusion, I think we should use the branding name